client/app/bundles/course/assessment/submission/components/CommentField.jsx

```diff
@@ -14,6 +15,15 @@ const translations = defineMessages({
     id: 'course.assessment.submission.commentField.comment',
     defaultMessage: 'Comment',
   },
+  commentDelayed: {
+    id: 'course.assessment.submission.commentField.commentDelayed',
+    defaultMessage: 'Delayed Comment',
+  },
+  commentDelayedDescription: {
+    id: 'course.assessment.submission.commentField.commentDelayedDescription',
+    defaultMessage:
+      'This comment will only be visible to students after this submission is published',
```

Comment:
I think in the perspective of the grader, `submission is published` may not make too much sense? Perhaps something like `after the grades are published`?

Reply:
Thanks for the review! sure i think it sounds more intuitive as well